kube_apiserver_metrics/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# CHANGELOG - Kube_apiserver_metrics
```

Comment:
Any chance we can rename to simply `kube_apiserver`?

Reply:
unfortunately, no. The kube_apiserver integration has 2 checks. The kube_apiserver to collect the events (it's a corecheck that we have been using ever since we released the DCA). And this one, `kube_apiserver_metrics` to collect the metrics. 
We have decided against merging them for the sake of simplicity